src/adagrams.js

```diff
@@ -1,15 +1,177 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const scoreChart = {
```

Comment:
I think we'd want to rename this something like `letterPool` or `availableLetters` to better reflect what it holds.